src/eval.jl

```diff
@@ -99,3 +99,21 @@ handle("eval-repl") do data
     end
   end
 end
+
+handle("docs") do data
+  @dynamic let Media.input = Editor()
+    result = withpath(getpath(data)) do
```

Comment:
Because I have no idea how the docsystem works and just copied that from above.
It's probably unnecessary, but I'm not sure.